functional/iak-idevid-register-with-certificates/test.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+HTTP_SERVER_PORT=8080
+AGENT_ID="d432fbb3-d2f1-4a97-9ef7-75bd81c00000"
+TPM2_OPENSSL="https://github.com/tpm2-software/tpm2-openssl/releases/download/1.2.0/tpm2-openssl-1.2.0.tar.gz"
```

Comment:
From my point of view it would be better, to not have hardcoded version of release, but to take latest one.

Reply:
We have discussed it and decided to stick with the "known to be working" version for now. We can revisit the decision later.